.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/DatePeriodParserChs.cs

```diff
@@ -1131,6 +1136,108 @@ private DateTimeResolutionResult ParseQuarter(string text, DateObject referenceD
             return ret;
         }
 
+        private DateTimeResolutionResult ParseDecade(string text, DateObject referenceDate)
+        {
+            var ret = new DateTimeResolutionResult();
+            int century = referenceDate.Year / 100;
+            int decade;
+            int beginYear;
+            int beginMonth = 1, endMonth = 12;
+            int beginDay = 1, endDay = 31;
+            int decadePass = 9;

```

Comment:
What does decadePass mean?

Reply:
For example, 1970s is 1970-01-01 to 1979-12-31. I need to add 9 to 1970 to get the end year.